app/assets/stylesheets/components/_buttons.scss

```diff
@@ -0,0 +1,69 @@
+.btn{
+    
+    align-items: center;
+    background: #fff;

```

Comment:
Add this color in `color.scss` file and call it like `$white`

Reply:
ok